src/tiled/mapdocument.h

```diff
@@ -93,17 +93,14 @@ class MapDocument : public Document
                              MapFormat *format,
                              QString *error = nullptr);
 
-    QString lastExportFileName() const;
-    void setLastExportFileName(const QString &fileName);
-
     MapFormat *readerFormat() const;
     void setReaderFormat(MapFormat *format);
 
     FileFormat *writerFormat() const override;
     void setWriterFormat(MapFormat *format);
 
-    MapFormat *exportFormat() const;
-    void setExportFormat(MapFormat *format);
+    MapFormat* exportFormat() const override;
```

Comment:
Coding style: please put `*` against the function name.

Reply:
Done